tests/integration/test_validators.py

```diff
@@ -211,6 +265,20 @@ def test_allof_required(self):
         ):
             validator.validate({"another_prop": "bla"})
 
+    @pytest.mark.xfail(reason="Issue #20")
+    def test_allof_nullable(self):
+        schema = {
+            "allOf": [
+                {
```

Comment:
This subschema is invalid (not nullable). That means the whole `allof` should be invalid isn't it?